### DISCUSSION TITLE
missing Tenant Management

### DISCUSSION THREAD

**Discussion by rolliecolby-co** [oqtane/oqtane.framework#1361] [Q&A]
I am following the Oqtane Module Development video tutorials.
When I downloaded v2.0.2, ran it and initialized the DB, in the Admin Dashboard, I do not see Tenant Management.  
Did I miss a step?  How do I get it added to the Admin Dashboard?

**Comment by leigh-pointer** [comment]
hi @rolliecolby-co 
In release 2.0.1 the Tenant Management was removed to reduce confusion.  [#1088 ](https://github.com/oqtane/oqtane.framework/pull/1088)

**Reply by rolliecolby-co** [reply]
makes sense; thank you for the reply, Leigh
